target_game.py

```diff
@@ -8,9 +8,12 @@
 

```

Comment:
Hii, so I think you have a really nice project and I am excited to play the game myself at some point! The readme file is very nicely detailed. I would think about maybe giving it a more unique name instead of target_shooting_game to make it a bit more exciting. Also, I would like to know just a tiny bit more about your exact implementation steps in the readme file, just so I get an overview of your code but in written words. Also, if you want you can add the target picture to your readme file by creating an issue (upload a picture there) and then copy the link of the issue and add it to the readme file. But that's just me personally and definitely not a must-have.

Reply:
Hey, thank you so much for your review! These are some great tips, I will definitely try to implement them :)